back/src/Guide/Api/Guider.hs

```diff
@@ -13,24 +14,33 @@ import Servant (Handler (..), ServantErr)
 import Servant.Server.Generic
 
 import Guide.Config (Config)
+import Guide.Api.Utils (RequestDetails)
+import Guide.State (DB)
 
 
 -- | Custom 'Guider' type holds the 'Config' always on hand.
```

Comment:
* It's not just `Config` anymore
* It would be good to mention that it's a `Handler`-like type

```suggestion
-- | A type for Guide handlers. Provides access to everything in 'ConfigHub'.
```